src/www/ui/change-license-processPost.php

```diff
@@ -95,10 +95,20 @@ function Output()
     $decisionMark = @$_POST['decisionMark'];
     if(!empty($decisionMark))
     {
-      $itemTableName = $this->uploadDao->getUploadtreeTableName($itemId);
-      /** @var ItemTreeBounds */
-      $itemTreeBounds = $this->uploadDao->getItemTreeBounds($itemId,$itemTableName);
-      $errMsg = $this->clearingDao->markDirectoryAsIrrelevant($itemTreeBounds,$groupId,$userId);
+      if(!is_array($itemId)) {
+        $itemTableName = $this->uploadDao->getUploadtreeTableName($itemId);
+        /** @var ItemTreeBounds */
+        $itemTreeBounds = $this->uploadDao->getItemTreeBounds($itemId,$itemTableName);
+        $errMsg = $this->clearingDao->markDirectoryAsIrrelevant($itemTreeBounds,$groupId,$userId);
+      }else{
+        $uploadTreeId = "";
+        foreach($itemId as $uploadTreeId){
+          $itemTableName = $this->uploadDao->getUploadtreeTableName($uploadTreeId);
```

Comment:
You could move the following three lines to a function, since they are duplicated to the other if-case above